rpmbuild/ec2rl.spec

```diff
@@ -44,17 +44,17 @@ rm -rf /usr/bin/ec2rl
 %dir
 
 %changelog
-* Tue December 8 2020 Greg Dunn <gregdunn@amazon.com> - 1.1.6
+* Tue Dec 8 2020 Greg Dunn <gregdunn@amazon.com> - 1.1.6
  - Update EC2 Rescue for Linux to version 1.1.6
-* Wed December 12 2018 Sean Poynter <seanpoyn@amazon.com - 1.1.5
+* Wed Dec 12 2018 Sean Poynter <seanpoyn@amazon.com - 1.1.5
  - Update EC2 Rescue for Linux to version 1.1.5
-* Wed August 22 2018 Sean Poynter <seanpoyn@amazon.com - 1.1.4
+* Wed Aug 22 2018 Sean Poynter <seanpoyn@amazon.com - 1.1.4
  - Update EC2 Rescue for Linux to version 1.1.4
-* Mon April 30 2018 Sean Poynter <seanpoyn@amazon.com - 1.1.3
+* Mon Apr 30 2018 Sean Poynter <seanpoyn@amazon.com - 1.1.3
  - Update EC2 Rescue for Linux to version 1.1.3
-* Thur April 05 2018 Sean Poynter <seanpoyn@amazon.com - 1.1.2
+* Thu Apr 05 2018 Sean Poynter <seanpoyn@amazon.com - 1.1.2
  - Update EC2 Rescue for Linux to version 1.1.2
-* Thur Feb 22 2018 Sean Poynter <seanpoyn@amazon.com - 1.1.1
```

Comment:
Is this necessary in the diff for this commit?

Reply:
Without changing the date format, the rpm would not build. I am unsure why dates in the changelog would cause the whole build to fail but once this change was made the build worked properly.

Reply:
But only some dates cause breakage? There's an unchanged date string on L59. I suppose it doesn't really matter (outside the context of the RPM build anyway) what format is used as long as it's consistent so be sure to update all the entries for consistency.

Reply:
> But only some dates cause breakage? There's an unchanged date string on L59. I suppose it doesn't really matter (outside the context of the RPM build anyway) what format is used as long as it's consistent so be sure to update all the entries for consistency.

Correct, only some dates caused breakage. Essentially, any date which did not have a 3-letter month abbreviation or did not have a 3-letter day abbreviation was a problem. Running "make rpm" would cause an error along the lines of "bad date in changelog".